src/components/tab/index.stories.tsx

```diff
@@ -82,6 +88,14 @@ export const VariantEnclosed: StoryType = {
   },
 };
 
+export const VariantFullWidth: StoryType = {
```

Comment:
I'm wondering what is the purpose of this const? I'm guessing its for storybook but I don't see it being used except for this initialisation.

Reply:
This const defines a story. More info here: https://storybook.js.org/docs/writing-stories/typescript#typing-stories-with-meta-and-storyobj

Reply:
Thanks :)